api/v1alpha3/awscluster_webhook.go

```diff
@@ -84,6 +84,11 @@ func (r *AWSCluster) ValidateUpdate(old runtime.Object) error {
 }
 
 func (r *AWSCluster) Default() {
+	// Default to allow open access to the bastion host if no CIDR Blocks have been set
+	if r.Spec.Bastion.CIDRBlocks == nil {
+		r.Spec.Bastion.CIDRBlocks = []string{"0.0.0.0/0"}
+	}
```

Comment:
Might be worth adding validation of the CIDRs to the webhook as well.

Reply:
Sure!  I can add that!

Reply:
added validation of the CIDR blocks. I used regex to validate. Let me know what you think